src/app/components/JsonLDGraph.tsx

```
@@ -0,0 +1,10 @@
+import type { Graph } from 'schema-dts';
+import 'server-only';
+
+export function JsonLDGraph({data}: {readonly data: Graph['@graph']}) {
+    // eslint-disable-next-line react/no-danger -- we're using JSON.stringify to set a <script> tag's contents; it's fine
+    return <script type="application/ld+json" dangerouslySetInnerHTML={{__html: JSON.stringify({
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)